testutil/MongoSuite.go

```diff
@@ -32,37 +32,37 @@ type MongoSuite struct {
 // will return the same DBServer until TearDownDBServer is called.  In most cases, calls to DBServer() are completely
 // unnecessary, as calls to DB() will automatically instantiate the DBServer if needed.  If a test suite calls
 // DBServer(), then TearDownDBServer() MUST be called in the TearDownSuite function (or TearDownTest, if appropriate).
-func (suite *MongoSuite) DBServer() *dbtest.DBServer {
-	suite.dbServerMutex.Lock()
-	defer suite.dbServerMutex.Unlock()
+func (m *MongoSuite) DBServer() *dbtest.DBServer {
```

Comment:
To be completely honest, that was me exploring the code. I completely forgot I had even done that.